src/gui/mainwindow.cpp

```diff
@@ -136,7 +136,12 @@ MainWindow::MainWindow(QWidget *parent)
 
     Preferences* const pref = Preferences::instance();
     m_uiLocked = pref->isUILocked();
-    setWindowTitle("qBittorrent " VERSION);
+    setWindowTitle("qBittorrent " VERSION
+#ifdef Q_OS_WIN64
+                   " (64 bit)"
```

Comment:
@okeatime:  I'm pretty sure you should use  `" (64-bit)"` instead.  It seems to be more conventional to use the hyphen (at least in most English programs).  

Thanks.